query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -1167,3 +1170,20 @@ func asSchemaMutationProcedureSpec(spec plan.ProcedureSpec) *universe.SchemaMuta
 	}
 	return spec.(*universe.SchemaMutationProcedureSpec)
 }
+
+type MergeFilterRule struct{}
+
+func (MergeFilterRule) Name() string {
+	return universe.MergeFiltersRule{}.Name()
```

Comment:
The `flux` version of the rule is `MergeFiltersRule`, where "filters" is plural, can you rename this rule for consistency's sake?